tests/benchmarks/micro_benchmarks/test_cublaslt_function.py

```diff
@@ -63,15 +63,15 @@ def test_cublaslt_gemm_command_generation(self):
         (benchmark_cls, _) = BenchmarkRegistry._BenchmarkRegistry__select_benchmark(self.benchmark_name, Platform.CUDA)
         benchmark = benchmark_cls(
             self.benchmark_name,
-            parameters='--batch 2:16:2 --shapes 2:4,4:8,8:32 32:128:4,128,128 --in_types fp16 fp32 fp64',
+            parameters='--batch 2:16:2 --shapes 2:4,4:8,8:32 32:128:4,128,128 --in_types fp16 fp32 fp64 int8',
         )
         self.assertTrue(benchmark._preprocess())
         self.assertEqual(4 * (2 * 2 * 3 + 2) * 3, len(benchmark._commands))
```

Comment:
do you need to update last 3 to 4?

Reply:
thanks a lot ! updated to use len(benchmarks._args.in_types)